src/lite-yt-embed.js

```diff
@@ -46,10 +46,23 @@ class LiteYTEmbed extends HTMLElement {
 
         this.addNoscriptIframe();
 
-        playBtnEl.removeAttribute('href');
+        // for the PE pattern, change anchor's semantics to button
+        if(playBtnEl.nodeName === 'A'){
+            playBtnEl.removeAttribute('href');
+            playBtnEl.setAttribute('tabindex', '0');
+            playBtnEl.setAttribute('role', 'button');
+            // fake button needs keyboard help
+            playBtnEl.addEventListener('keypress', e => {
```

Comment:
how about keydown and using `e.key` instead?

Reply:
Oh, yeah that's a good call. Old muscle memory here :) 

I'll make that change now, friend.